File: WebService/SnapTown.Models/Town.cs
using System;
using System.Collections.Generic;
using System.Data.Entity.Spatial;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SnapTown.Models
{
    public class Town
    {
        public int TownID { get; set; }
        public string Name { get; set; }
        public string OtherSpellings { get; set; }
        public DbGeography Location { get; set; }

        public int CountryID { get; set; }

        public virtual Country Country { get; set; }
        public virtual ICollection<Subscription> Subscriptions { get; set; }
    }
}

File: Android/SnapTown/src/com/example/snaptown/models/Town.java
package com.example.snaptown.models;

public class Town {
	public int townId;
	public String name;
	public String country;
	public boolean isSubscribed;

	public Town(int townId, String name, String country, boolean isSubscribed) {
		this.townId = townId;
		this.name = name;
		this.country = country;
		this.isSubscribed = isSubscribed;
	}

	public CharSequence getName() {
		return name;
	}

	public CharSequence getCountry() {
		return country;
	}

}
